tests/chainer_tests/functions_tests/math_tests/test_polygamma.py

```diff
@@ -3,79 +3,58 @@
 import numpy
 
 import chainer
-from chainer.backends import cuda
 import chainer.functions as F
-from chainer import gradient_check
 from chainer import testing
-from chainer.testing import attr
 
 
 @testing.parameterize(*testing.product({
     'shape': [(), (3, 2)],
     'dtype': [numpy.float16, numpy.float32, numpy.float64]
 }))
 @testing.with_requires('scipy')
```

Comment:
Could you write this decorator innermost? Otherwise you'll get a TypeError trying to apply it after backend injection.

Reply:
@hvy  Done.

Reply:
Thanks!